src/Util/extractCloudinaryProps.js

```diff
@@ -3,7 +3,7 @@ import {Transformation, Util} from 'cloudinary-core';
 const CLOUDINARY_REACT_PROPS = {includeOwnBody: true};
 
 // Map Cloudinary props from array to object for efficient lookup
-const CLOUDINARY_PROPS = Transformation.PARAM_NAMES.map(Util.camelCase).reduce(
```

Comment:
Needed to add those since this function is now also used by Image component,
Prevents transformation params + publicId + breakpoints to become <img> attributes.